var/spack/repos/builtin/packages/flex/package.py

```diff
@@ -61,3 +65,18 @@ def url_for_version(self, version):
             url += "/archive/flex-{0}.tar.gz".format(version.dashed)
 
         return url
+
+    @run_after('install')
+    def symlink_lex(self):
+        if self.spec.satisfies('+lex'):
+            with working_dir(self.prefix.bin):
+                if (os.path.isfile('flex') and not
+                   os.path.lexists('lex')):
+                    symlink('flex', 'lex')
+            with working_dir(self.prefix.lib):
+                if (os.path.isfile('libfl.a') and not
+                   os.path.lexists('libl.a')):
+                    symlink('libfl.a', 'libl.a')
+                if (os.path.isfile('libfl.so') and not
```

Comment:
Not at the moment. This is one of the major things we're working on for the next spack release.